charts/rhio/templates/cm.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "app.fullname" . }}-config
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+data:
+  config.yaml: |-
+  {{- .Values.configuration | .Files.Get | nindent 4 }}
```

Comment:
@ktatarnikovhiro here is where I'm trying to get the configuration file (the path is set in `values.yaml`) into the template. Currently when I run `helm template rhio --debug --values ./rhio/values.yaml --set configuration=config.yaml` to test the template build locally the output correctly includes the values contained in `config.yaml` file, but when I run `helm upgrade --install rhio rhio-charts/rhio --values ./rhio/values.yaml --set configuration=config.yaml --dry-run` it does not. I think this is a misunderstanding on my part about how dynamic values can be injected into the templates. However I can't see another way to do this. Any advise much appreciated.

Reply:
From the example you shared https://github.com/glaciation-heu/kg_exporter/blob/main/charts/app/values.yaml I can see that the values are actually defined directly in the `values.yaml` file and not passed in from a separate config file as we would like to do.